scripts/tables.sql

```diff
@@ -177,4 +175,4 @@ INSERT INTO migrations (name)
 SELECT '202307111517030_add_subscription_invitations_table'
 WHERE NOT EXISTS (
     SELECT 1 FROM migrations WHERE name = '202307111517030_add_subscription_invitations_table'
-);
\ No newline at end of file
+);
```

Comment:
You would also need to change the name of the last migration line 175 and line 177